docs/project/coding-style.md

```diff
@@ -52,7 +52,7 @@ program code and documentation code.
 
 * Use existing Docker test files (`name_test.go`) for inspiration. 
 
-* Run <a href="../test-and-docs" target="_blank">the full test suite</a> on your
+* Run [the full test suite](test-and-docs.md) on your
```

Comment:
Yep

I searched for that, and there's actually a configuration in Hugo to automatically add `target blank` to "external" (`http(s)`) links;

- https://github.com/spf13/hugo/commit/483fc8fa3dd8db7d399ef5ed05a3b7f42ea19152
- https://github.com/spf13/hugo/commit/a509a23255d20769621a2d2a714964fc3c6bca55